packages/react/src/components/Accordion/Accordion.tsx

```diff
@@ -85,6 +86,11 @@ const Accordion = ({
   const { className: triggerClassName, ...triggerProps } = trigger.props;
 
   const elementId = useId();
+  const prefersReducedMotionEnabled = prefersReducedMotion();
+
+  if (prefersReducedMotionEnabled) {
+    animationTiming = false;
+  }
```

Comment:
When `animationTiming` is falsy, the `ExpandCollapse` component disables the animation for expanding/collapsing the panel.